.travis.yml

```diff
@@ -29,16 +29,18 @@ addons:
     - libboost1.55-all-dev
     - libboost1.55-tools-dev
     - g++-4.8
+    - cmake
 
 install:
-  - 'if [ $TRAVIS_OS_NAME == "linux" ]; then echo "using gcc : : ccache g++-4.8 : <cflags>-std=c11 <cxxflags>-std=c++11 ;" > ~/user-config.jam; fi'
+  - 'if [ $TRAVIS_OS_NAME == "linux" ]; then echo "using gcc : : ccache g++-4.8 : <cflags>-std=c11 <cxxflags>-std=c++11 ;" > ~/user-config.jam; export CXX="g++-4.8" CXXFLAGS="-std=c++11"; fi'
   - 'if [ $TRAVIS_OS_NAME == "osx" ]; then echo "using darwin : : ccache clang++ : <cflags>-std=c11 <cxxflags>-std=c++11 <compileflags>-Wno-deprecated-declarations ;" > ~/user-config.jam; fi'
-  - if [ $TRAVIS_OS_NAME == "osx" ]; then brew update > /dev/null && brew install --quiet ccache boost-build; fi
+  - if [ $TRAVIS_OS_NAME == "osx" ]; then brew update > /dev/null && brew install --quiet ccache boost-build && brew upgrade --quiet cmake ; fi
   - ccache -V && ccache --show-stats && ccache --zero-stats
 
 script:
 # until we have a version of boost-build that's new enough, disable warnings
 # as errors
   - bjam --hash -j3 variant=$variant -l900 warnings-as-errors=off $toolset $target
+  - mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=$variant .. && make -j3 && make test
```

Comment:
I thought you had to say ``-G "Unix Makefiles"`` to ``cmake`` for it to generate makefiles.